src/main/java/org/myrobotlab/service/interfaces/MotorControl.java

```diff
@@ -115,7 +115,7 @@
 	 */
 	void setInverted(boolean invert);
 
-	void setPowerLevel(double power);
+	// void setPowerLevel(double power);

```

Comment:
maybe keep this around and just deprecate it?  you can delicate to the prefered moveTo method.

Reply:
setPowerLevel did the "calculation" of data in the MotorControl - which is incorrect. 
The calculation of doing what a MotorController needs to do to provide the final data sent to the hardware is ONLY in the MotorController..

The MotorControl only provides its data for the MotorController to operate on.
As a MotorControl I want to control a motor with the move functions, this function is a MotorController function